test/unit/spec/VerifyWebauthn_spec.js

```diff
@@ -14,7 +14,7 @@ define([
   'sandbox',
   'util/webauthn',
   'helpers/xhr/MFA_REQUIRED_allFactors',
-  'helpers/xhr/MFA_REQUIRED_multipleFactors',
+  'helpers/xhr/MFA_REQUIRED_multipleWebauthn_question',
```

Comment:
can we also rename the variable we use? It's just easier in the future to know what the mock response is if the name matches the actual mock response